bench-templates/src/lib.rs

```diff
@@ -30,8 +30,10 @@ pub fn bench_pcs_method<
         &PCS::VerifierKey,
         usize,
         fn(usize, &mut ChaCha20Rng) -> P,
+        fn(usize, &mut ChaCha20Rng) -> P::Point,
```

Comment:
Since we're adding quite a few arguments here, some comments would be helpful =)

Reply:
Done.
However, I feel like this `bench-templates` can be improved. It is exclusive to multilinear polynomials at the moment. (Since `max_degree` is set to `num_vars`)

Reply:
A gentle reminder on this @Pratyush